frontend/packages/ceph-storage-plugin/src/components/block-pool/body.tsx

```diff
@@ -54,10 +61,11 @@ export type BlockPoolStatusProps = {
   status: string;
   name?: string;
   error?: string;
+  isClusterReady?: boolean;
 };
 
 export const BlockPoolBody = (props: BlockPoolBodyPros) => {
```

Comment:
```suggestion
export const BlockPoolBody:React.FC<BlockPoolBodyProps> = (props) => {
```
I don't know how this slipped from original PR. Let's fix it now.